### PR TITLE
Upgrade spring to 2.7.x

### PR DESCRIPTION
**Pull request by zhem0004**
<h3>What does this PR do?</h3>
This PR upgrades version of spring to 2.7.x.

<h3>Motivation</h3>
The newer version is more up to date and there are fixes that have been applied in Spring Boot 2.7

<h3>To fix</h3>
The presentation is not being set as current
Insert document request is not working properly
Presentation is not displayed properly

image

### CODE REVIEW

**Comment by gustavotrott** [comment]
It would be better to avoid changing the name of the configs!
cause it will break the servers that have changed the config using the old names!
I would say to keep using the old name e.g `beans.presentationService.defaultUploadedPresentation` instead of `defaultUploadedPresentation`.

It seems we can keep the old name and make the `resources.xml` read it!
image(11)

**Review by gustavotrott — CHANGES_REQUESTED** [review]
Keep the previous names for the configs to avoid break issues!

File: bigbluebutton-web/grails-app/conf/bigbluebutton.properties [hunk 1]
```diff
@@ -419,7 +417,10 @@ endWhenNoModeratorDelayInMinutes=1
 # List of features to disable (comma-separated)
 # Available options: breakoutRooms, captions, chat, externalVideos, layouts, learningDashboard, polls, screenshare,
 # sharedNotes, virtualBackgrounds, downloadPresentationWithAnnotations, importPresentationWithAnnotationsFromBreakoutRooms
-#disabledFeatures=
+disabledFeatures=none
+
+# Notify users that recording is on
+notifyRecordingIsOn=false
```
Comment (gustavotrott) [inline bigbluebutton-web/grails-app/conf/bigbluebutton.properties:424]:
```suggestion
```

Duplicated!

File: bigbluebutton-web/grails-app/conf/bigbluebutton.properties [hunk 2]
```diff
@@ -377,7 +375,7 @@ presentationBaseURL=${bigbluebutton.web.serverURL}/bigbluebutton/presentation
 beans.presentationService.presentationBaseUrl=${presentationBaseURL}
 #----------------------------------------------------
 # Inject values into grails service beans
-beans.presentationService.presentationDir=${presentationDir}
+#presentationDir=${presentationDir}
```
Comment (gustavotrott) [inline bigbluebutton-web/grails-app/conf/bigbluebutton.properties:378]: Prefer removing it instead of comment! If will not be necessary anymore.

**Review by gustavotrott — CHANGES_REQUESTED** [review]
After your change the `securitySalt` is being read from `bigbluebutton.properties` instead of `/etc/bigbluebutton/bbb-web.properties`.

All configs set on `/etc/bigbluebutton/bbb-web.properties` have priority! Make sure to keep this behavior!

File: bigbluebutton-web/grails-app/conf/bigbluebutton.properties (hunk 2 above)
Comment (zhem0004) [reply bigbluebutton-web/grails-app/conf/bigbluebutton.properties]: Set to be as it used to

**Pull request merged**